chapters/infrastructure/linters/reading/README.md

```diff
@@ -1,14 +1,37 @@
-### Linters
+# Linters
 
-Owner: Gabi Mocanu
-Reviewer: Alex Apostolescu
+Linters are tools for checking consistency and quality of code, but they can be used on every type of content.
+Enforce a consistent style for writing content, making it easier to read and maintain for content creators.
+There are different types of linters, depending on the type of content.
+For example, a linting tool for code is different from a linting tool for text.
 
-definition
+Right now, we are using [`super-linter`](https://github.com/super-linter/super-linter).
+`Super-linter` is a combination of various linters that can be used locally or in CI/CD pipelines.
 
-roles
+## Usage
 
-types of linters
+### Locally
 
-using linters locally
+There are multiples ways to run linters locally.
 
-integrating linters in the submission process (of drills and projects) and in the contribution process, via GitHub workflows
+`Super-linter` can be run locally inside a Docker container using this [script](https://gist.github.com/gabrielmocanu/9606cfd789e07b0a1650d0b349fd15f2).
+You can see that you will need two arguments, the path to the codebase and the path to the linting rules.
+Linting rules are used because sometimes the linter might be too strict.
+Configuring these rules can ignore some errors that are not relevant for the project.
+The linting rules that now we are using and recommending are inside the [actions](https://github.com/open-education-hub/actions) repository.
```

Comment:
```suggestion
The linting rules that we are currently using and recommending are inside the [actions](https://github.com/open-education-hub/actions) repository.
```

Reply:
Maybe point directly to the super linter rules https://github.com/open-education-hub/actions/tree/main/super-linter/rules